test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -8423,6 +8423,50 @@ def forward(self, x, y):
                       input_names=['input_x', 'input_y'],
                       dynamic_axes={'input_x': [0, 1, 2, 3], 'input_y': [0, 1, 2, 3]})
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_index_add_normal(self):
+        class M(torch.nn.Module):
```

Comment:
Thanks! Can you add a couple tests and use index_add_ inside a loop or if block? Just to make sure the nested blocks can handle inplace updates from this op.